domain/src/main/java/org/oppia/domain/topic/TopicListController.kt

```diff
@@ -49,6 +49,7 @@ const val TEST_TOPIC_ID_0 = "test_topic_id_0"
 const val TEST_TOPIC_ID_1 = "test_topic_id_1"
 const val FRACTIONS_TOPIC_ID = "GJ2rLXRKD5hw"
 const val RATIOS_TOPIC_ID = "omzF4oqgeTXd"
+const val EQUAL_PARTS_STORY_ID = "wANbh4oOClga"
```

Comment:
Do you mean matthew goes to the bakery? That’s the name of the story. Equal parts is the name of the exploration.

Reply:
Done. Updated to `MATTHEW_GOES_TO_THE_BAKERY_STORY_ID`